### PR TITLE
Add jsfunctionality to simon says game to enable sounds and add color-pies

### PR DESCRIPTION
**Pull request by lumodon**

### CODE REVIEW

File: public/javascripts/controls.js [hunk 1]
```diff
@@ -0,0 +1,81 @@
+  'use strict'
```
Comment (jason00111) [inline public/javascripts/controls.js]: The name 'javascripts' is funny

File: pbcopy [hunk 2]
```diff
@@ -0,0 +1 @@
+/Users/helen/desktop/LGfolder/simonsays
```
Comment (jason00111) [inline pbcopy]: What's up with this?

File: public/javascripts/controls.js [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+  //array to hold played color-notes
+  const playedNotes = []
+  //array of set of color-notes
+  const notesToPlay = []
```
Comment (jason00111) [inline public/javascripts/controls.js]: The tabbing is off on lines 1 through 5

File: public/javascripts/controls.js [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+const resetVars = () => {
+  //reset variables
+}
+//main game loop
+while(gameRunning) {
+  playedNotes.push("cyan", "indigo", "orange", "cyan", "brown", "indigo", "orange", "indigo")
+  timeOutLoop(playedNotes.length, playBack, 1000)
+  //getInPut()
+
+  const soundsToNotes = {
```
Comment (jason00111) [inline public/javascripts/controls.js]: Maybe `colorsToNotes`?

File: public/javascripts/controls.js [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+    resetVars()
+    console.log("You win!")
+  }
+
+  //run playback function
+  function timeOutLoop(reps, fn, delay) {
+    if(reps > 0) {
+      const timerId = setTimeout(() => {
+        fn(reps)
+        timeOutLoop(reps - 1, fn, delay)
```
Comment (jason00111) [inline public/javascripts/controls.js]: Nice recursion!

File: public/javascripts/controls.js [hunk 6]
```diff
@@ -0,0 +1,81 @@
...
+  }
+
+  if(playedNotes.length === 20) {
+    gameRunning = false
+    resetVars()
+    console.log("You win!")
+  }
+
+  //run playback function
+  function timeOutLoop(reps, fn, delay) {
```
Comment (jason00111) [inline public/javascripts/controls.js]: Maybe a more descriptive name for `fn`?

File: pbcopy (hunk 2 above)
Comment (lumodon) [inline pbcopy]: Whoops, used "pwd > pbcopy" instead of "pwd | pbcopy"

**Pull request merged**